dataset/dictionary/organizations.ts

```diff
@@ -645,6 +653,22 @@ export default [
     pronunciationJa: "しもつきのこ",
     tags: [ "nodkrai", "organization" ],
   },
+  {
+    en: "Hyperborea",
+    ja: "ヒュペルボレイア",
+    zhCN: "亥珀波瑞亚",
+    zhTW: "亥珀波瑞亞",
+    notes: "テイワット大陸の北方にかつて存在していた王国",
+    tags: [ "nodkrai", "organization" ],
```

Comment:
There is no best tag for the nations in the Teyvat history now, but considering the consistency with the existing nations (Natlan, Fontaine, etc.), `location` tag might be a better fit.

```suggestion
    tags: [ "nodkrai", "location" ],
```